xdsl_torch/utils/import_program.py

```diff
@@ -1,13 +1,70 @@
+from itertools import zip_longest
+from typing import Any
+
 import torch
 from xdsl.builder import ImplicitBuilder
-from xdsl.dialects import func
-from xdsl.dialects.builtin import TensorType
+from xdsl.dialects import arith, func
+from xdsl.dialects.builtin import BoolAttr, FloatAttr, IntegerAttr, TensorType
 from xdsl.ir import SSAValue
 
 from xdsl_torch.dialects.torch_mapping import XDSL_TORCH_OPS
 from xdsl_torch.utils.type_mapping import TORCH_DTYPE_TO_XDSL_TYPE
 
 
+def literal_to_ssa(value: Any) -> tuple[str, SSAValue]:
+    """
+    Construct a ConstantOp for a scalar value.
```

Comment:
One thing that is missing and likely to trip me up is that this assumes that there is an implicit builder active when this function is called. TBH I would get rid of the implicit builder here, it's more useful as a little DSL to make a snippet of IR more readable, but gets a little confusing when the operation being created is not within sight of the ImplicitBuilder call.